### PR TITLE
Fix protected swapchain images

### PR DESCRIPTION
**Pull request by sfricke-samsung**
Currently the `VkImage`s from a Swapchain with `VK_SWAPCHAIN_CREATE_PROTECTED_BIT_KHR` were not being marked as protected and throwing a false error for `VUID-vkCmdDraw-commandBuffer-02712`

Simple 1 line fix and added a test to show a case it was failing prior

### CODE REVIEW

File: tests/vkpositivelayertests.cpp [hunk 1]
```diff
@@ -11719,6 +11719,183 @@ TEST_F(VkPositiveLayerTest, InitSwapchain) {
...
+    // Without device to test with, blocking this test from non-Android platforms for now
+    printf("%s VK_KHR_surface_protected_capabilities test logic not implemented, skipping test for non-Android\n", kSkipPrefix);
+    return;
+#endif
+
+    m_errorMonitor->ExpectSuccess();
+    SetTargetApiVersion(VK_API_VERSION_1_1);
+
+    if (!AddSurfaceInstanceExtension()) {
+        printf("%s surface extensions not supported, skipping CmdCopySwapchainImage test\n", kSkipPrefix);
```
Comment (ncesario-lunarg) [inline tests/vkpositivelayertests.cpp:11738]: `CmdCopySwapchainImage` -> `ProtectedSwapchainImageColorAttachment`?

File: tests/vkpositivelayertests.cpp [hunk 2]
```diff
@@ -11719,6 +11719,183 @@ TEST_F(VkPositiveLayerTest, InitSwapchain) {
...
+        m_instance_extension_names.push_back(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
+    } else {
+        printf("%s Did not find required instance extension %s; skipped.\n", kSkipPrefix,
+               VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
+        return;
+    }
+    ASSERT_NO_FATAL_FAILURE(InitFramework(m_errorMonitor));
+
+    if (!AddSwapchainDeviceExtension()) {
+        printf("%s swapchain extensions not supported, skipping CmdCopySwapchainImage test\n", kSkipPrefix);
```
Comment (ncesario-lunarg) [inline tests/vkpositivelayertests.cpp:11752]: `CmdCopySwapchainImage` -> `ProtectedSwapchainImageColorAttachment`?

**Review by ncesario-lunarg — APPROVED** [review]
LGTM! Minor nits in the comments.

**Comment by ncesario-lunarg** [comment]
Running through [internal CI](http://erusea:4040/8767).

File: tests/vkpositivelayertests.cpp (hunk 2 above)
Comment (sfricke-samsung) [reply tests/vkpositivelayertests.cpp]: 100% copy and paste error

**Pull request merged**